server/routes/approvingLicences/handlers/comDetails.test.ts

```diff
@@ -0,0 +1,47 @@
+import { Request, Response } from 'express'
+
+import ComDetailsRoutes from './comDetails'
+import CommunityService from '../../../services/communityService'
+
+const communityService = new CommunityService(null, null) as jest.Mocked<CommunityService>
+
+describe('Route Handlers - COM Details', () => {
+  const handler = new ComDetailsRoutes(communityService)
+  let req: Request
+  let res: Response
+
+  beforeEach(() => {
+    res = {
+      render: jest.fn(),
+      locals: {
+        licence: {
+          comUsername: 'jrogan',
+        },
+      },
+    } as unknown as Response
+
+    communityService.getStaffDetailByUsername = jest.fn()
+  })
+
+  describe('GET', () => {
+    it('should render com details view', async () => {
+      communityService.getStaffDetailByUsername.mockResolvedValue({
+        staff: {
+          forenames: 'Joe',
```

Comment:
Might be worth considering whether this is a bit of a controversial test subject (or may be in the future?)

Perhaps consider a more generic example?